src/main/java/school/hei/haapi/service/aws/FileService.java

```diff
@@ -62,4 +63,16 @@ public File createTempFile(byte[] bytes) {
       throw new ApiException(SERVER_EXCEPTION, ioException.getMessage());
     }
   }
+
+  public File createTempFile(MultipartFile file) {
+    File tempFile;
+    try {
+      tempFile = File.createTempFile("file", "temp");
```

Comment:
temp file name should be with uuid to make it more random, also, discuss with @Iloniavo concerning this method